File: result/translate/starchat/java2cpp/c++/FIND_ONE_EXTRA_CHARACTER_STRING_1.cpp
char f_gold ( string strA, string strB ) {
  int res = 0, i ;
  for ( i = 0 ;
  i < strA. length ( ) ;
  i ++ ) {
    res ^= strA. at ( i ) ;
  }
  for ( i = 0 ;
  i < strB. length ( ) ;
  i ++ ) {
    res ^= strB. at ( i ) ;
  }
  return ( ( char ) ( res ) ) ;
}

File: result/translate/transcoder/java2python/python/FIND_ONE_EXTRA_CHARACTER_STRING_1.py
def f_gold ( strA , strB ) :
    res , i = 0 , 0
    for i in range ( len ( strA ) ) :
        res ^= strA [ i ]
    for i in range ( len ( strB ) ) :
        res ^= strB [ i ]
    return ( chr ( res ) , i )
